server/pairing/preflight/preflight.go

```diff
@@ -40,7 +38,7 @@ func preflightHandler(w http.ResponseWriter, r *http.Request) {
 
 func makeCert(address net.IP) (*tls.Certificate, []byte, error) {
 	now := timesource.GetCurrentTime()
-	log.Debug("makeCert", "system time", time.Now().String(), "timesource time", now.String())
+	logutils.ZapLogger().Debug("makeCert", zap.Stringer("system time", time.Now()), zap.Stringer("timesource time", now))
```

Comment:
I'd suggest to create a custom zap field like this:
```
func UTCTime(key string, t time.Time) zap.Field {
	return zap.Time(key, t.UTC())
}
```
and use that instead of `zap.Time("someField", someTime)`.
Also, maybe this is a good time to consider whether we want the logged times as a string, or as an epoch, i.e.:
```go
func UTCTime(key string, t time.Time) zap.Field {
	return zap.String(key, fmt.Sprintf("%d", t.UTC().Unix()))  // Or Millis or Nanos
}
```
Having the time as an epoch instead of the default string format of `zap.Time` perhaps is more useful for debugging

Reply:
Thanks @richard-ramos , I like the approach. Adapted code accordingly.